src/routes/transactions/mappers/common/native-staking.mapper.ts

```diff
@@ -327,20 +333,23 @@ export class NativeStakingMapper {
    * Each {@link KilnDecoder.KilnPublicKeyLength} characters represent a validator to withdraw,
    * and each native staking validator has a fixed amount of 32 ETH to withdraw.
    *
-   * @param data - the decoded data of the transaction
-   * @param chain - the chain where the native staking deployment lives
+   * @param dataDecoded - the decoded data of the transaction
+   * @param chainId - the ID of the chain where the native staking deployment lives
+   * @param safeAddress - the Safe staking
    * @returns the rewards to withdraw from the native staking deployment
    */
-  private async getRewardsFromDataDecoded(
-    data: DataDecoded,
-    chain: Chain,
-  ): Promise<number> {
-    const publicKeys = this.getPublicKeysFromDataDecoded(data);
+  private async getRewardsFromDataDecoded(args: {
+    dataDecoded: DataDecoded;
+    chainId: string;
```

Comment:
Question: Why is `chain` changed to `chainId` here?

Reply:
Only `chain.chainId` is used. Passing the entire `Chain` seemed excessive. Should I revert it?

Reply:
All good.
In terms of memory and performance, there shouldn't be any significant differences. JavaScript is pass-by-value, but when dealing with objects, the variable holds a reference to the object rather than the object itself.